tests/grouping_test.py

```diff
@@ -15,13 +15,22 @@
     ('1+2', '3-4', _Mul(_Add(1, 2), _Add(3, _Mul(-1, 4))))
 ]
 
-modifiable_delimiter_pairs = {
+example_relation_lists = [
+    ('x=1', 'y=2', [_Eq(x, 1), _Eq(y, 2)])
+]
+
+func_modifiable_delimiter_pairs = {
     '(': ')',
-    '\\lgroup': '\\rgroup',
+    '\\lgroup ': '\\rgroup ',
```

Comment:
spaces intentional here?

Reply:
@lily514 yeah most latex commands get spaces after them, was just making this match